portafly/src/i18n/locales/en/shared.yml

```diff
@@ -128,3 +128,19 @@ loading:
   title_description: Title displayed as part of the loading state
   aria_label: Loading spinner  #-----TODO: revise this aria label
   aria_label_description: Aria label for the Loading component
+
+data_list_filter:
+  text_input:
+    aria_label: Filter text input
+    aria_label_description: Aria label for the filter text input
+    placeholder: "Filter by {{category}}"
+    placeholder_description: Filter by text input. This applies for filters by group/organization and admin
```

Comment:
```suggestion
    typeahead_filter_description: Filter by typeahead a text input. This applies for filters by group/organization and admin
```

Reply:
Why not call placeholder? 🤔

Reply:
Because `placeholder` is not a microcopy element, as listed in the [3scale Microcopy guidelines](https://docs.google.com/document/d/1LBkuJdItYpdwa3-ewLp7gqGsj0VJxDWhp4w1ijKBqdc/edit#heading=h.9qe5nbyh73of)

Reply:
I see